test/unit/footprints.js

```diff
@@ -148,9 +148,9 @@ describe('promicro', function() {
         (fp_line (start 15.24 -8.89) (end -17.78 -8.89) (layer F.SilkS) (width 0.15))
         (fp_line (start -17.78 -8.89) (end -17.78 8.89) (layer F.SilkS) (width 0.15))
 
-        (fp_line (start -15.24 -6.35) (end -12.70 -6.35) (layer F.SilkS) (width 0.15))
-        (fp_line (start -15.24 -6.35) (end -15.24 -8.89) (layer F.SilkS) (width 0.15))
+        (fp_line (start -12.70 -6.35) (end -15.24 -6.35) (layer F.SilkS) (width 0.15))
         (fp_line (start -12.70 -6.35) (end -12.70 -8.89) (layer F.SilkS) (width 0.15))
+        (fp_line (start -15.24 -6.35) (end -15.24 -8.89) (layer F.SilkS) (width 0.15))
```

Comment:
@mrzealot note that this is exactly the same as removed line 152.

Reply:
@mrzealot the reason we have these two changes when `orientation: up` is that we now mirror the primitive on the `y` axis.  It is the exact same geometry as before.  Merely the order in which we draw the individual lines is slightly modified.